rundetection/rules/tosca_rules.py

```diff
@@ -0,0 +1,45 @@
+"""Rules for TOSCA"""
+import logging
+from copy import deepcopy
+from pathlib import Path
+from typing import List
+
+from rundetection.ingest import JobRequest, get_run_title
+from rundetection.rules.rule import Rule
+
+logger = logging.getLogger(__name__)
+
+
+class ToscaStitchRule(Rule[bool]):
+    """
+    Rule for stitching TOSCA Runs
+    """
+
+    @staticmethod
+    def _is_title_similar(title: str, other_title: str) -> bool:
+        return title[:-4] == other_title[:-4]
```

Comment:
Could we do this using Difflib and a SequenceMatcher? then if it is close enough by the equivalent of 4 chars difference or better then assume it's the same?